src/@next/Skeleton/components/SkeletonImage/SkeletonImageSquare.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import { StyledSkeletonImageSquare } from './SkeletonImageStyle';
+
+export interface SkeletonImageSquareProps {
+  height: string;
+  width: string;
+}
+
+export const SkeletonImageSquare = ({
+  height,
```

Comment:
should we have default width and height?

Reply:
agree! asked Rebecca for a default. this will have default height=80px and width=80px